src/Model/Behavior/TrashBehavior.php

```diff
@@ -27,7 +27,7 @@
  */
 class TrashBehavior extends Behavior
 {
-    public const SOFT_DELETE_OPTION = 'muffin.trash.soft-delete';
+    public const DELETE_OPTION_NAME = 'trash;
```

Comment:
```suggestion
    public const DELETE_OPTION_NAME = 'trash';
```
Closing quote was missing.

Reply:
Nvm, I missed the follow up.